Cargo.toml

```diff
@@ -202,6 +206,7 @@ try-runtime = [ "polkadot-cli/try-runtime" ]
 fast-runtime = [ "polkadot-cli/fast-runtime" ]
 runtime-metrics = [ "polkadot-cli/runtime-metrics" ]
 pyroscope = ["polkadot-cli/pyroscope"]
+jemalloc-stats = ["polkadot-node-core-pvf/jemalloc-stats", "polkadot-overseer/jemalloc-stats"]
```

Comment:
Please rename this to `jemalloc` or `jemalloc-allocator` and then do the trick that @koute mentioned with enabling it automatically for Linux.

Reply:
Yeah, i've done this. Thanks!! Please review again